src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js

```diff
@@ -58,8 +58,8 @@ function CalCalendarImportController(
 
   function submit() {
     var notificationMessages = {
-      progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      progressing: 'Submitting importing calendar request',
```

Comment:
It's a convention that we add "..." to the end of the message if it's a message about something that's being done (for example "Loading...", "Submitting...").
So I think you should keep the "..." dots.